owa/app/js/subscription/subscription.controller.js

```diff
@@ -5,10 +5,13 @@ class SubscriptionController {
     $rootScope.links["Open Concept Lab"] = "";
     $rootScope.links["Subscription"] = "subscription";
 
+    var oclApiUrl = "api.openconceptlab.org"
```

Comment:
Are these constants or variables that are to be changed?

Reply:
constant, for variable I would have used [let](https://www.w3schools.com/js/js_let.asp) keyword

Reply:
Is the above a constant?

Reply:
It's a constant.
for variable I would have used let keyword

Reply:
If i later on assign a value to `oclApiUrl`, will the compiler complain?